### PR TITLE
docs: softened dependency on tests, small edits

### PR DESCRIPTION
**Pull request by ptrdvrk**
Updates of README.md:
 - moved the link to Discord above the first gif
 - softened dependency on tests
 - added a link to a Py tutorial
 - added links to remote recording instructions for appmap clients
 - moved "contact us" to the top of the troubleshooting section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+
 ![AppMap diagrams](https://vscode-appmap.s3.us-east-2.amazonaws.com/media/002.gif "AppMap diagrams")
 
 ## Summary of features
 
 - UML-inspired Dependency Map that displays key application components and how they are interrelated during execution 
 - Execution Trace diagrams that visualize code and data flows
-- List of Webservices generated automatically from test cases
-- List of SQL queries generated automatically from test cases
+- List of Webservices generated automatically from exeduted code
```
Comment (kgilpin) [inline README.md:20]: Typo here

File: README.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+- List of SQL queries generated automatically from executed code
```
Comment (kgilpin) [inline README.md:21]: "List of" is a noun but "Search for" is a verb.

File: README.md [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
 - Search for class, package or function
 - Code linkage of the diagram to the source code it relates to
```
Comment (kgilpin) [inline README.md:23]: "it relates to" seems redundant

File: README.md [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
+Get accurate software design diagrams drawn automatically in your IDE. In less than 2 minutes you can go from installing this extension to viewing diagrams in your IDE, because AppMap creates diagrams automatically when your code is executed. AppMap helps you:
 
 - Onboard to code architecture, with no extra work for the team 
 - Conduct code and design reviews using live and accurate data
-- Troubleshoot hard-to-understand bugs using a "top-down" approach.
+- Troubleshoot hard-to-understand bugs using a "top-down" approach
 
 Each interactive diagram links directly to the source code, and the information is easy to share.
 
+**Join us on [Discord](https://discord.com/invite/N9VUap6) or [GitHub](https://github.com/applandinc/vscode-appland).**
```
Comment (kgilpin) [inline README.md:11]: I thought we were going to put some user quotes in here? I think that would add a lot of validation to this page.

File: README.md (hunk 1 above)
Comment (ptrdvrk) [reply README.md:20]: Doh. :( Thank you!

File: README.md (hunk 4 above)
Comment (ptrdvrk) [reply README.md:11]: Elizabeth asked for moving the Discord link there. I'll prepare another PR with user quotes shortly.

File: README.md (hunk 2 above)
Comment (ptrdvrk) [reply README.md:21]: How about this:

- UML-inspired Dependency Map that displays key application components and how they are interrelated during execution 
- Execution Trace diagrams that visualize code and data flows
- List of Web services generated automatically from executed code
- List of SQL queries generated automatically from executed code
- Code linkage of the diagram to the source code
- Filtering by class, package or function

File: README.md (hunk 3 above)
Comment (ptrdvrk) [reply README.md:23]: Noted.

**Review by kgilpin — APPROVED** [review]

**Pull request merged**